README.md

```diff
@@ -121,8 +121,8 @@ The General flow is described in the diagram:
 The changes to the driver are almost purely changing the search strategy it uses to find the extension .dll file.
 It is very reliant on nuget to place things correctly, so it's worth calling out the specific contracts:
 
-1. dotnet driver expects that a restore of an extensions package will have the unpacked package directory (`NUGET_PACKAGE_CACHE/packages/dotnet-extension-package/1.0.0`) copied to the dotnet-extensions package hive (`NUGET_PACKAGE_CACHE/dotnet-extensions/dotnet-extension-package/1.0.0`)
-2. dotnet driver expects that a project.lock.json for the `dotnet-extension-package` will exist in the dotnet-extensions package hive (`NUGET_PACKAGE_CACHE/dotnet-extensions/dotnet-extension-package/1.0.0/project.lock.json`)
+1. dotnet driver expects that a restore of an extensions package will have the unpacked package directory (`~/.nuget/packages/dotnet-extension-package/1.0.0`) copied to the dotnet-extensions package hive (`~/.nuget/dotnet-extensions/dotnet-extension-package/1.0.0`)
```

Comment:
~ does refer to home, and yes that is how it should be if it wasn't configured otherwise through NUGET_PACKAGES or nuget.config. You want to start reading it in the CLI.

You can add a footnote saying the ~/.nuget/Packages can be redirected and that is outside the scope of this document, you want to add however that the extensions folder can be redirected separately as well with a TBD mechanism (or suggest one)